posthog/api/survey.py

```diff
@@ -801,6 +801,19 @@ class Meta:
         fields = ["survey_config"]
 
 
+class SurveyAPIActionSerializer(serializers.ModelSerializer):
```

Comment:
that was added in https://github.com/PostHog/posthog/pull/26759/ already
probably just need to merge master into this branch?